backend/src/main/java/com/benine/backend/http/EditPresetHandler.java

```diff
@@ -43,23 +43,27 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
           throws IOException, ServletException {
     
     try {
-      String overwriteTag = request.getParameter("overwritetag");
-      String overwritePosition = request.getParameter("overwriteposition");
+      final String overwriteTag = request.getParameter("overwritetag");
+      final String overwritePosition = request.getParameter("overwriteposition");
+      final String overwriteName = request.getParameter("overwritename");
       int presetID = Integer.parseInt(request.getParameter("presetid"));
       String tags = request.getParameter("tags");
+      String name = request.getParameter("name");
       
-      Preset preset = getPresetController().getPresetById(presetID);   
+      Preset preset = getPresetController().getPresetById(presetID);
       Set<String> tagList = new HashSet<>();
       if (tags != null) {
         tagList = new HashSet<>(Arrays.asList(tags.split("\\s*,\\s*"))); 
       }
       if (overwriteTag.equals("true")) {
         updateTag(preset, tagList);
       }
-      
       if (overwritePosition.equals("true")) {
         updatePosition(preset);
       }
+      if (overwriteName.equals("true")) {
```

Comment:
I  don't think we need the overwrite parameter, you could make simply this distinction by checking if the name parameter is null or not.